src/Resources/contao/templates/survey/survey_question_multiplechoice.html5

```diff
@@ -69,8 +69,8 @@
             <div><input type="radio" name="<?php echo $this->ctrl_name; ?>" id="ctrl_<?php echo $this->ctrl_id; ?>_<?php echo $counter; ?>" class="radio<?php echo $this->ctrl_class; ?>" value="<?php echo $counter; ?>"<?php if (($this->values['value'] ?? 0) == $counter): ?> checked="checked"<?php endif; ?> /> <label for="ctrl_<?php echo $this->ctrl_id; ?>_<?php echo $counter; ?>"><?php echo Contao\StringUtil::specialchars($choice); ?></label></div>
             <?php $counter++; ?>
         <?php endforeach; ?>
-        <?php if ($this->blnOther && isset($this->values['value'])): ?>
-            <div><input type="radio" name="<?php echo $this->ctrl_name; ?>" id="ctrl_<?php echo $this->ctrl_id; ?>_<?php echo $counter; ?>" class="radio<?php echo $this->ctrl_class; ?>" value="<?php echo $counter; ?>"<?php if ($this->values['value'] == $counter): ?> checked="checked"<?php endif; ?> /> <label for="ctrl_<?php echo $this->ctrl_id; ?>_<?php echo $counter; ?>"><?php echo $this->otherTitle; ?></label> <input type="text" name="other_<?php echo $this->ctrl_name; ?>" class="text<?php echo $this->ctrl_class; ?>" <?php if (strlen($this->values["other"])): ?>value="<?php echo Contao\StringUtil::specialchars($this->values['other']); ?>" <?php endif; ?>/></div>
+        <?php if ($this->blnOther): ?>
+            <div><input type="radio" name="<?php echo $this->ctrl_name; ?>" id="ctrl_<?php echo $this->ctrl_id; ?>_<?php echo $counter; ?>" class="radio<?php echo $this->ctrl_class; ?>" value="<?php echo $counter; ?>"<?php if (isset($this->values['value']) && ($this->values['value'] == $counter)): ?> checked="checked"<?php endif; ?> /> <label for="ctrl_<?php echo $this->ctrl_id; ?>_<?php echo $counter; ?>"><?php echo $this->otherTitle; ?></label> <input type="text" name="other_<?php echo $this->ctrl_name; ?>" class="text<?php echo $this->ctrl_class; ?>" <?php if (strlen($this->values["other"])): ?>value="<?php echo Contao\StringUtil::specialchars($this->values['other']); ?>" <?php endif; ?>/></div>
```

Comment:
If this template is rendered at the BE with act=list, then $this->values['value'] leads to an exception, because $this->values is undefined (here NULL). This is due to the loss of the widget context, which is not given at the BE in act=list. The problem would have to be solved globally.

Reply:
Oh, I just forgot an additional check here (i just fixed the first one and oversaw the second). If we let the check where it was before, we won't see any "other"-field output. This was the reason I changed it.

Reply:
Fixed in [96b12f2](https://github.com/pdir/contao-survey/pull/31/commits/96b12f2b2a39290b2237cdff84c0cc78dd872596)